strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/TrashController.java

```diff
@@ -124,15 +119,14 @@ public ResponseEntity delete()
     @PreAuthorize("hasAuthority('MANAGEMENT_UNDELETE_TRASH')")
     @RequestMapping(value = "{storageId}/{repositoryId}/{path:.+}",
                     method = RequestMethod.POST,
-                    produces = MediaType.TEXT_PLAIN_VALUE)
+                    produces = MediaType.ALL_VALUE)
```

Comment:
I don't understand the necessity of this change.

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/MediaType.html

 `MediaType.TEXT_PLAIN_VALUE` is still supported.

Reply:
+1. Me neither.